cmd/neofs-cli/modules/root.go

```
@@ -107,7 +109,23 @@ func init() {
 
 	// Cobra also supports local flags, which will only run
 	// when this action is called directly.
-	// rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	rootCmd.Flags().Bool("version", false, "Application version and NeoFS API compatibility")
+}
+
+func entryPoint(cmd *cobra.Command, _ []string) {
+	printVersion, err := cmd.Flags().GetBool("version")
+	if err == nil && printVersion {
```

Looks like we can ignore error
```
printVersion, _ := cmd.Flags().GetBool("version")
if printVersion {}
```